java/Anton-Kozyk/src/com/zoolatech/lecture10/tasks/_1/FileReader.java

```diff
@@ -0,0 +1,39 @@
+package com.zoolatech.lecture10.tasks._1;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Date;
+
+public class FileReader {
+    private File directory;
+
+    public FileReader(File directory) {
+        this.directory = directory;
+    }
+
+    public void directoryContent() {
+        System.out.println("Directory content:");
+        String[] list = directory.list();
+        for (String file : list) {
+            if (!file.equals("error.log")) {
+                System.out.println("-" + file);
+            }
+        }
+    }
+
+    public void fileContent(File file) throws IOException {
+        if (!file.exists()) {
+            System.err.println("File '" + file.getName() + "' not found. Try again...\n");
+            try (FileWriter fileWriter = new FileWriter(directory.getPath() + "/error.log", true)) {
+                fileWriter.write(file.getName() + " [" + new Date() + "]\n");
```

Comment:
Please use java 8 date-time classes. util.Date is an old approach. More context here https://programminghints.com/2017/05/still-using-java-util-date-dont/

Reply:
Done, now it uses LocalDateTime.
Thank you, that's very useful information for me! And thanks for the link.